Jenkinsfile

```diff
@@ -31,3 +42,37 @@ node('linux') {
 			sh 'make test'
 	}
 }
+
+def git_commit = ''
+def go_list = ''
+
+stage 'Go & Git Info'
+node('linux') {
+
+	def gopath = gopath_linux()
+	def workdir = workdir_linux(gopath)
+
+	dir(workdir) {
+		checkout scm
+
+		// HACK: Recommended approach for getting command output is writing to and then reading a file.
+		sh 'mkdir -p tmp'
+		sh 'go list . > tmp/GO_LIST'
```

Comment:
@vdice thought about this a bunch. does the PR builder provide any information on the PR number and/or forked repo/branch? if so, that might be enough? otherwise, how exactly are we getting the SDK code in question?

Regarding the bundling job, that sounds like a good option if you can tell the workflow-cli job to replace some code in the `vendor/` directory...